libs/membership/membership-common/src/main/kotlin/net/corda/membership/lib/MemberInfoExtension.kt

```diff
@@ -137,6 +137,17 @@ class MemberInfoExtension {
         const val NOTARY_KEY_HASH = "corda.notary.keys.%s.hash"
         const val NOTARY_KEY_SPEC = "corda.notary.keys.%s.signature.spec"
 
+        /**
+         * Interop role name
+         */
+        const val INTEROP_ROLE = "interop"
+
+        /**
+         * Interop role properties
```

Comment:
Should this be something more descriptive like:
```suggestion
         * Interop service name
```
(I'm not sure what interop service name is so maybe there's a better description)